### PR TITLE
Add ReviewsList file, add ReviewTile file, rename Ratings_Reviews to App. 

### PR DESCRIPTION
**Pull request by jordanvillacorta**
Added placeholders for my #1  and #2 Reviews in my Review List. 
Can someone review this and make sure I'm not missing anything?

Thanks!

### CODE REVIEW

File: client/src/ratings_reviews/App.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReviewsList from './ReviewsList.jsx';
+
+class RatingsReviews extends React.Component {
```
Comment (SebasCC99) [inline client/src/ratings_reviews/App.jsx:4]: Please change RatingsReviews to a functional component, the props are the current product id, so there is no need to hold duplicate data in state

File: client/src/ratings_reviews/ReviewsList.jsx [hunk 2]
```diff
@@ -1,20 +1,16 @@
 import React from 'react';
+// import Rating from 'react-rating'; --> This is for later
+import ReviewTile from './ReviewTile.jsx';
 
-class ReviewsList extends React.Component {
```
Comment (SebasCC99) [inline client/src/ratings_reviews/ReviewsList.jsx:3]: Please also change component to a functional component, you are getting the data as props, no need to have duplicate data as `state`

File: client/src/ratings_reviews/ReviewTile.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+let ReviewTile = (props) => {
```
Comment (SebasCC99) [inline client/src/ratings_reviews/ReviewTile.jsx:3]: change `let ReviewTile` to `const ReviewTile`

**Review by SebasCC99 — CHANGES_REQUESTED** [review]
Please take a look at the comments

File: client/src/ratings_reviews/ReviewTile.jsx (hunk 3 above)
Comment (jordanvillacorta) [reply client/src/ratings_reviews/ReviewTile.jsx]: Change made!

File: client/src/ratings_reviews/App.jsx (hunk 1 above)
Comment (jordanvillacorta) [reply client/src/ratings_reviews/App.jsx]: Change made!

**Review by SebasCC99 — APPROVED** [review]

**Pull request merged**